_includes/manuals/2.1/1.2_release_notes.md

```diff
@@ -389,6 +389,9 @@ By default it is deployed using [systemd socket activation](http://0pointer.de/b
 
 ### Deprecations
 
+* Ability to trigger puppet run for hosts from Foreman is deprecated and will be removed in version 2.2 ([more info](https://community.theforeman.org/t/dropping-run-puppet-button-with-a-replacement-using-remote-execution-plugin/12781/25)). The same functionality can be achieved using the Remote Execution plugin. There are two options:
+    * There is `Puppet run once - SSH default` job template that takes `puppet_options` as parameter before it's invoked
+    * Another option is for `mcollective`/`choria` users. The best way is write own job template that leverages `load_hosts` macro for choosing hosts and then this selection is used as a parameter for `mco` program, for ie. `mco rpc puppetd runonce -I <%= load_hosts(search: input('hosts')).join(' ') %>`
```

Comment:
```suggestion
    * Another option is for `mcollective`/`choria` users. The best way is to write your own job template that leverages `load_hosts` macro for choosing hosts and use it as a parameter for `mco` program, for example: `mco rpc puppetd runonce -I <%= load_hosts(search: input('hosts')).join(' ') %>`. This template would be executed against a host that has `mco` and access to the target hosts, instead of being executed at the target hosts directly.
```